mod_reforged/hooks/skills/effects/berserker_rage_effect.nut

```diff
@@ -0,0 +1,46 @@
+::mods_hookExactClass("skills/effects/berserker_rage_effect", function (o) {
+	local create = o.create;
+	o.create = function()
+	{
+		create();
+        this.m.Description = "This character is in a berserking rage."
+	}
+
+	o.getTooltip <- function()
+	{
+		local ret = this.skill.getTooltip();
+		ret.extend([
+			{
+				id = 10,
+				type = "text",
+                icon = "ui/icons/special.png",
+				text = "Receives only " + ::MSU.Text.colorGreen((this.calcDamageTakenMult() * 100.0) + "%") + " of any damage"
+			},
+			{
+				id = 11,
+				type = "text",
+				icon = "ui/icons/regular_damage.png",
+				text = ::MSU.Text.colorGreen("+" + this.m.RageStacks + "%") + " Damage"
+			},
+			{
+				id = 12,
+				type = "text",
+				icon = "ui/icons/bravery.png",
+				text = ::MSU.Text.colorGreen("+" + this.m.RageStacks + "%") + " Resolve"
+			},
+			{
+				id = 13,
+				type = "text",
+				icon = "ui/icons/initiative.png",
+				text = ::MSU.Text.colorGreen("+" + this.m.RageStacks + "%") + " Initiative"
+			}
+		]);
+		return ret;
+	}
+
+	// New function replicating the current hard-coded vanilla calculation. Changing this will currently only affect the tooltip
```

Comment:
We should consider overwriting the vanilla `onUpdate` function to now refer to this function, so changing this affects both the tooltip and the behavior.

Reply:
That should be postponed to the PR which actually wants to change how berserker rage works